contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -53,7 +53,7 @@ All libraries are reviewed by the community before becoming part of Boost. There
 
 If you have a deep interest in a particular library that for some reason no longer has an active owner or maintainer, or is being less well maintained than might be ideal (find this out by asking questions on the https://lists.boost.org/mailman/listinfo.cgi/boost[Boost developers' mailing list]), then consider requesting, again via the mailing list, to become a library maintainer. For larger libraries there can certainly be more than one maintainer. 
```

Comment:
It's worth talking to the current maintainer listed in `meta.json` file before.

Reply:
added that point